data_loading.py

```diff
@@ -145,11 +145,11 @@ def get_variables():
         # "shr_energy_tot_cali","selected","n_showers_contained",  # only if CC0piNp variables are saved!
         # We do not want to load "vector" variables into the final dataframe, as they take up 
         # a lot of memory
-        # "pfp_generation_v",
+        "pfp_generation_v",
         "shr_energy_cali",
-        # "trk_dir_x_v",
-        # "trk_dir_y_v",
-        # "trk_dir_z_v",
+        "trk_dir_x_v",
+        "trk_dir_y_v",
+        "trk_dir_z_v",
```

Comment:
These variables ending in "_v" are going to take up a lot of space in the dataframe, because they are rugged arrays. If they are necessary for intermediate steps, please load them temporarily in the `process_` functions.